Gems/EMotionFX/Code/EMotionFX/Pipeline/SceneAPIExt/Behaviors/RootMotionExtractionRuleBehavior.cpp

```diff
@@ -43,6 +43,12 @@ namespace EMotionFX::Pipeline::Behavior
         }
 
         Rule::RootMotionExtractionRule* rootMotionExtractionRule = azrtti_cast<Rule::RootMotionExtractionRule*>(&target);
+        auto data = rootMotionExtractionRule->GetData();
+        if (!data)
+        {
+            return;
```

Comment:
is it expected that root motion rules don't exist or should there be a warning?

Reply:
you can have empty data, that just mean you haven't apply the root motion extraction modifier. You could also have a rule but set the data to null from c++ api. Totally valid cases here, simple return should do.